widget/check.go

```diff
@@ -200,6 +201,21 @@ func NewCheck(label string, changed func(bool)) *Check {
 	return c
 }
 
+// NewCheckWithData returns a check widget connected with the specified data source.
+func NewCheckWithData(label string, data binding.Bool) *Check {
+	check := NewCheck(label, func(b bool) {
+		data.Set(b)
+	})
+	check.Checked = data.Get()
```

Comment:
Not necessary if bindings notify new listeners of current state

Reply:
Resolved